### PR TITLE
 save the startedAt and finishedAt properties as a Date when saving to the database

### PR DESCRIPTION
**Pull request by EvadLhorg**
save the startedAt and finishedAt properties as a Date when saving to the database

cast the input parameters of getAllGames to a date so the comparison works

update stats.js so the local db is used as config.js got removed some time ago

add number of unique players to the stats result

### CODE REVIEW

File: server/stats.js [hunk 1]
```diff
@@ -4,9 +4,8 @@ const _ = require('underscore');
 const monk = require('monk');
 
 const GameService = require('./services/GameService.js');
-const config = require('./config.js');
 
-let db = monk(config.dbPath);
+let db = monk('mongodb://127.0.0.1:27017/throneteki');
```
Comment (ystros) [inline server/stats.js:8]: Changes in this file seem unrelated (perhaps stuff you've done for local dev?)

**Review by ystros — CHANGES_REQUESTED** [review]

File: server/stats.js (hunk 1 above)
Comment (EvadLhorg) [reply server/stats.js:8]: Kind of unrelated, i guess a seperate PR is a good idea. I would like an addition to the stats script to get the number of unique players (see line 139 to 142) and this change is necessary as the config.js file got removed some time ago

**Comment by EvadLhorg** [comment]
fixes #3062

File: server/stats.js (hunk 1 above)
Comment (EvadLhorg) [reply server/stats.js:8]: Will revert these changes and open up a seperate PR with the usage of configservice, thanks!
